plugins/api-docs/src/components/AsyncApiDefinitionWidget/AsyncApiDefinition.tsx

```diff
@@ -142,6 +142,28 @@ const useStyles = makeStyles((theme: BackstageTheme) => ({
   },
 }));
 
+const fetchResolver = {
+  order: 199,
+  canReadValue: '^https?://[^\\s$.?#].[^\\s]*$',
+  canRead(file: any) {
+    if ((this.canReadValue as any) instanceof Boolean) {
+      return this.canReadValue;
+    }
+    return file.url.match(new RegExp(this.canReadValue as unknown as string));
+  },
```

Comment:
I think this entire thing and `canReadValue` can be replaced by `canRead: /^https?:\/\//`, or perhaps the full regex if needed?

Reply:
Changed to your suggestion 🚀